gslib/command.py

```diff
@@ -1929,6 +1953,61 @@ def _ApplyThreads(self, thread_count, process_count, recursive_apply_level,
         # block the semaphore on a WorkerThread completion.
         worker_semaphore.release()
 
+  def _GetGcloudStorageArgs(self, sub_opts, gsutil_args, gcloud_storage_map):
+    if gcloud_storage_map is None:
+      raise GcloudStorageTranslationError(
+          'Command "{}" cannot be translated to gcloud storage because the'
+          ' translation mapping is missing.'.format(self.command_name))
+    args = []
+    if isinstance(gcloud_storage_map.gcloud_command, str):
+      args.append(gcloud_storage_map.gcloud_command)
+    elif isinstance(gcloud_storage_map.gcloud_command, dict):
+      # If a command has sub-commands, e.g pap set, pap get.
```

Comment:
Should this be all-caps "PAP"?

Reply:
I think this should be lowercase as I am talking about the command itself. I have added gsutil in front of it to make it explicit that it is a command that I am talking about.